service/src/test/java/org/mskcc/cmo/metadb/service/MetadbServiceTest.java

```diff
@@ -207,5 +210,24 @@ public void testSampleHistoryListIsAscendingByImportDate() throws Exception {
         List<SampleMetadata> sampleMetadataHistory = sampleService.getSampleMetadataHistoryByIgoId(igoId);
         Assertions.assertThat(sampleMetadataHistory).isSorted();
     }
+    
+    @Test
+    public void testFindPatientByPatientAlias() {
+        Assertions.assertThat(patientRepository.findPatientByPatientAlias("C-MP789JR")).isNotNull();
+    }
+    
+    @Test
+    public void testFindPatientByPatientAliasWithExpectedFailure() {
+        MetaDbPatient patient = new MetaDbPatient();
```

Comment:
related but not a change that is needed here:

- add a patient service that returns an updated metadb patient obj

Reply:
sample service would use the new patientService instead of the patientRepository directly